index.html

```diff
@@ -622,38 +622,9 @@ <h3 class="text-primary mb-4">Newsletter</h3>
     <!-- Template Javascript -->
     <script src="js/main.js"></script>
 
-    <script>
```

Comment:
This code has been moved in a separate file `auth.js` with some optimisations.

Reply:
@muhafiz5814 do what you want.